typescript-sdk/CHANGELOG.md

```
@@ -23,6 +23,52 @@
+* **deps:** bump @opentelemetry/sdk-trace-base from 2.1.0 to 2.7.1 in /typescript-sdk ([#3738](https://github.com/langwatch/langwatch/issues/3738)) ([4e725e6](https://github.com/langwatch/langwatch/commit/4e725e662b2366c6e7d87fc80a8e95990703aba5))
+* **deps:** bump @opentelemetry/sdk-trace-base in /typescript-sdk ([4e725e6](https://github.com/langwatch/langwatch/commit/4e725e662b2366c6e7d87fc80a8e95990703aba5))
+* **deps:** bump ora from 9.3.0 to 9.4.0 in /typescript-sdk ([#3728](https://github.com/langwatch/langwatch/issues/3728)) ([7e0fec2](https://github.com/langwatch/langwatch/commit/7e0fec295f51e72149911acd54fa6828732d211a))
+* **deps:** bump the npm_and_yarn group across 1 directory with 4 updates ([#3609](https://github.com/langwatch/langwatch/issues/3609)) ([8c609a8](https://github.com/langwatch/langwatch/commit/8c609a899b884534a9fe6914aed250381547f824))
+* **main:** release typescript-sdk 0.27.0 ([#3624](https://github.com/langwatch/langwatch/issues/3624)) ([00abe7e](https://github.com/langwatch/langwatch/commit/00abe7e9ca1d5139e403ea2a73b0f4ce9f7512e6))
+* release as 3.2.1 ([ca9d7a9](https://github.com/langwatch/langwatch/commit/ca9d7a9231a7b3d9d8cf9a28a48fa494b1daeb4b))
+* release as 3.2.1 (override release-please from 3.3.0) ([#3615](https://github.com/langwatch/langwatch/issues/3615)) ([ca9d7a9](https://github.com/langwatch/langwatch/commit/ca9d7a9231a7b3d9d8cf9a28a48fa494b1daeb4b))
+* scope Release-As to langwatch, restore other components ([e259e79](https://github.com/langwatch/langwatch/commit/e259e796b50e4d060e5c7f42cad1927f1da8a83d))
+* **typescript-sdk:** bind 2 SDK-level [@unimplemented](https://github.com/unimplemented) prompt-tag scenarios ([#3458](https://github.com/langwatch/langwatch/issues/3458)) ([#3696](https://github.com/langwatch/langwatch/issues/3696)) ([73b9afa](https://github.com/langwatch/langwatch/commit/73b9afa9ce01fd202b0e93aa4e36f552e386d11c))
+* **typescript-sdk:** single-footer shadow Release-As 0.27.0 ([a48b3eb](https://github.com/langwatch/langwatch/commit/a48b3eb2cdf7d355b56136a11f5092484b7ab169))
+
+## [0.27.0](https://github.com/langwatch/langwatch/compare/typescript-sdk@v0.27.0...typescript-sdk@v0.27.0) (2026-05-13)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)